### ISSUE TITLE
Windows 10 1903不能启动

### ISSUE THREAD

**Issue by abcfy2** [aliyun/oss-browser#217]
运行`oss-browser.exe`什么反应都没有，删除重装也一样，也没有生成什么日志文件。

但是我用electron示例项目直接启动是没有任何问题的，是否考虑下升级electron重新build一次？

**Comment by Ari1c** [comment]
请问您是在win10 上下载64位最新版本来解压使用的吗

**Comment by abcfy2** [comment]
对。是从1809升级而来的1903版本，无法运行程序。我前天重装了最新版本的1903，似乎又能打开了。疑似Electron版本太老，很多新的操作系统支持都有问题，已经提交了PR升级了Electron，目前在我这边都正常

**Comment by Ari1c** [comment]
我们这边当时electron是低版本开发，有机会会更新electron版本的到时会反馈给你
